aries-site/src/examples/templates/forms/ShippingExample.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Main,
+  MaskedInput,
+  Select,
+  Text,
+  TextInput,
+} from 'grommet';
+
+import { FormContainer } from '.';
+import { emailMask, emailValidation, phoneMask, states } from './formHelpers';
+
+export const ShippingExample = () => {
+  const [formValues, setFormValues] = React.useState({});
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Heading level={3} margin="none">
+            Shipping
+          </Heading>
+          <Text>for your HPE products</Text>
+        </Header>
+        <Main
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
+            validate="blur"
+            value={formValues}
+            onChange={setFormValues}
+            onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+          >
+            <Box>
+              <Heading level={4} margin={{ bottom: 'small' }}>
+                Shipping Information
+              </Heading>
+              <Text>Shipping Address</Text>
+              <FormField htmlFor="firstName" name="firstName">
```

Comment:
This is an issue on grommet side I think, but the formfield is not associating the label with the input field for any of these.

Reply:
Actually, none of these have labels specified 😬

Reply:
I will fix.

Reply:
Mmm then makes sense why no label is being found haha -- if labels aren't being displayed, we need to think about how accessibility can still be met. I can do some reading. I've read some things about having label still in the DOM but visually hidden.

Using placeholder text in place of a label doesn't meet accessibility on its own.

I don't know if this should stop the PR because it falls in line more with your other form discussion/question of how we design accessible forms.

Reply:
@taysea This [commit](https://github.com/hpe-design/design-system/pull/469/commits/cf52c7e3832507b1754a0aaec726c8c4796b3c86) proposes an approach for having field labels visually hidden, yet still available to screen readers. It implements this [suggestion from W3C](https://www.w3.org/WAI/tutorials/forms/labels/#note-on-hiding-elements) through a styled component.